pom.xml

```diff
@@ -42,7 +42,7 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
             <version>31.0.1-jre</version>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
```suggestion
```

Reply:
This one probably doesn't matter so much.

Reply:
google dependency (compile) is required to include one package that is used in the class FileSystemService
import com.google.common.collect.ImmutableMap;

Reply:
It might have previously been used in unit tests which is probably why it was added to the test scope. Let's leave it for now. Just be aware that if guava stuff is used in a unit test that this scope value should probably be removed.